designsafe/apps/workspace/api/views.py

```diff
@@ -471,6 +475,7 @@ def get(self, request, *args, **kwargs):
                         {
                             "app_id": "jupyter-lab-hpc",
                             "app_type": "tapis",
+                            "bundle_category": "Analysis",
```

Comment:
Is this a `bundle_category` that was missing? I see this line in `feat/Tapis-v3-redesign` already… oh because this was merged. Hm, okay. Moving on.